src/actions/request/creators.js

```diff
@@ -43,6 +43,26 @@ export function postContent(formName) {
   };
 }
 
+export function saveImage(image) {
+  return {
+    type: t.SEND_REQUEST,
+    payload: {
+      method: 'post',
+      dataType: 'images',
+      contentType: 'file',
+      requestData: image,
+      successDispatch: [saveImageSuccess],
+    }
+  };
+}
+
+export function saveImageSuccess(data) {
+  return {
+    type: t.SAVE_IMAGE_SUCCESS,
+    payload: data.fileName,
```

Comment:
Maybe it's better to be consistent and use an object as payload: `payload: data`